src/CMakeLists.txt

```diff
@@ -139,6 +143,7 @@ SET(LMMS_REQUIRED_LIBS
 	${PULSEAUDIO_LIBRARIES}
 	${JACK_LIBRARIES}
 	${OGGVORBIS_LIBRARIES}
+	${LAME_LIBRARIES}
```

Comment:
So the way CMake works on all of these libraries, the `LAME_LIBRARIES` variable will be a value of [`LAME_LIBRARIES-NOTFOUND`](https://cmake.org/cmake/help/ctest2.6docs.html#command:find_file) which will abort the build.  This is a problem with our build scripts and not necessarily a problem with this PR, but once merged, people without the proper lame libraries will get build errors, hence this comment about the "trick" mentioned here.  https://github.com/LMMS/lmms/pull/3615#issuecomment-306523077